RecoEcal/EgammaCoreTools/test/testEcalClusterTools.cc

```diff
@@ -73,13 +76,8 @@ void testEcalClusterTools::analyze(const edm::Event& ev, const edm::EventSetup&
   ev.getByToken(endcapClusterToken_, pEEClusters);
   const reco::BasicClusterCollection* eeClusters = pEEClusters.product();
 
-  edm::ESHandle<CaloGeometry> pGeometry;
-  es.get<CaloGeometryRecord>().get(pGeometry);
-  const CaloGeometry* geometry = pGeometry.product();
-
-  edm::ESHandle<CaloTopology> pTopology;
-  es.get<CaloTopologyRecord>().get(pTopology);
-  const CaloTopology* topology = pTopology.product();
+  const auto* geometry = &es.getData(geometryToken_);
```

Comment:
isnt this supposed to be this?
```suggestion
  const auto& geometry = es.getData(geometryToken_);
```
(my point is that it's not supposed to be a pointer after all, right?)

Reply:
The `ClusterTools::` functions below all take pointers as input so instead of changing all those to use `&geometry` I did it like this.

Reply:
I see, ok. And so you ran your calibration tool and it works well like this, right?

Reply:
We have been using this already in #35077 and it seems to run fine in the test configuration as well.